datafusion/physical-expr/src/aggregate/min_max.rs

```diff
@@ -453,6 +454,14 @@ fn min_batch(values: &ArrayRef) -> Result<ScalarValue> {
         DataType::LargeUtf8 => {
             typed_min_max_batch_string!(values, LargeStringArray, LargeUtf8, min_string)
         }
+        DataType::Utf8View => {
+            typed_min_max_batch_string!(
```

Comment:
Presumably this is using the fast kernel from https://github.com/apache/arrow-rs/pull/6053

Reply:
yes